src/rgw/setup/rgw.py

```diff
@@ -544,7 +544,18 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
             #    Log.info(f'User creation is successful on "{Rgw._machine_id}" node.')
             #    Rgw._set_consul_kv(rgw_consul_idx, const.CONSUL_LOCK_KEY, const.ADMIN_USER_CREATED)
             # else:
-            machine_ids = Rgw._get_cortx_conf(conf, const.MACHINE_IDS_KEY)
+            node_id=Rgw._machine_id
+            storage_set = Rgw._get_cortx_conf(conf,\
+                f'node>{node_id}>storage_set')
+            storage_set_count = Rgw._get_cortx_conf(conf,\
+                'cluster>storage_set_count')
+            machine_ids = []
+            for i in range(0, storage_set_count):
```

Comment:
Instead of 'i',  can we  give some meaningful name to the iterator like (count, index, storage_set_index, s_set_index)

Reply:
updated in recent patch